pkg/verify/testdata/keyrings/combined.txt

```diff
@@ -0,0 +1,56 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v2.0.22 (GNU/Linux)
```

Comment:
I don't see this line in the actual container being created (though I do see the "Comment" line)

Reply:
Differen't GnuPG versions?  :man_shrugging: